test/powershell/Modules/Microsoft.PowerShell.Utility/Format-List.Tests.ps1

```diff
@@ -8,63 +8,63 @@ Describe "Format-List" -Tags "CI" {
     }
 
     It "Should call format list without error" {
-        { $in | Format-List } | Should Not BeNullOrEmpty
+        { $in | Format-List } | Should -Not -BeNullOrEmpty
     }
 
     It "Should be able to call the alias" {
-        { $in | fl } | Should Not BeNullOrEmpty
+        { $in | fl } | Should -Not -BeNullOrEmpty
     }
 
     It "Should have the same output whether choosing alias or not" {
         $expected = $in | Format-List | Out-String
         $actual   = $in | fl          | Out-String
 
-        $actual | Should Be $expected
+        $actual | Should -Be $expected
     }
 
     It "Should produce the expected output" {
         $expected = "${nl}${nl}testName : testValue${nl}${nl}${nl}${nl}"
         $in = New-Object PSObject
         Add-Member -InputObject $in -MemberType NoteProperty -Name testName -Value testValue
 
-        $in | Format-List                  | Should Not BeNullOrEmpty
-        $in | Format-List   | Out-String   | Should Not BeNullOrEmpty
-        $in | Format-List   | Out-String   | Should Be $expected
+        $in | Format-List                  | Should -Not -BeNullOrEmpty
+        $in | Format-List   | Out-String   | Should -Not -BeNullOrEmpty
+        $in | Format-List   | Out-String   | Should -Be $expected
     }
 
     It "Should be able to call a property of the piped input" {
         # Tested on two input commands to verify functionality.
-        { Get-Command | Select-Object -First 5 | Format-List -Property Name }        | Should Not BeNullOrEmpty
+        { Get-Command | Select-Object -First 5 | Format-List -Property Name }        | Should -Not -BeNullOrEmpty
```

Comment:
Extra whitespace on the end between the closing brace and pipe seems unnecessary